src/Components/Login.js

```diff
@@ -1,53 +1,104 @@
+import React from "react"
+import Avatar from "@material-ui/core/Avatar"
+import Button from "@material-ui/core/Button"
+import CssBaseline from "@material-ui/core/CssBaseline"
+import TextField from "@material-ui/core/TextField"
+import LockOutlinedIcon from "@material-ui/icons/LockOutlined"
+import Typography from "@material-ui/core/Typography"
```

Comment:
Why are we importing these as a separate component rather than like this 
import {
  Button,
  Grid,
  TextField,
  Typography,
  withStyles
} from "@material-ui/core"

Reply:
Done